hydroflow_macro/build.rs

```diff
@@ -44,7 +44,9 @@ fn main() -> Result<(), Box<dyn Error>> {
         file!().replace(std::path::MAIN_SEPARATOR, "/")
     )?;
     writeln!(write, "{}", PREFIX)?;
-    for op in OPERATORS {
+    let mut ops = OPERATORS.to_vec();
+    ops.sort_by_key(|op| op.name);
```

Comment:
not a big deal but would be more efficient to remove the `#[derive(Clone)]` and just have:
```rust
    let mut ops: Vec<_> = OPERATORS.iter().collect();
    ops.sort_by_key(|op| op.name);
```
(The difference is that this way we sort a vec of references instead of a vec of cloned owned values)

Reply:
Huh I thought I tried and failed at that, but it works! Thanks.